libethereum/BlockChain.cpp

```diff
@@ -950,8 +950,8 @@ ImportRoute BlockChain::insertBlockAndExtras( VerifiedBlockRef const& _block,
     }
     pieceUsageBytes += writeSize;
 
-    LOG( m_loggerInfo ) << "Block " << tbi.number() << " DB usage is " << writeSize
-                        << ". Piece DB usage is " << pieceUsageBytes << " bytes";
+    LOG( m_loggerDetail ) << "BLOCK_DB+" << writeSize << ". Piece DB usage is " << pieceUsageBytes
```

Comment:
this information may be used by qa team or monitoring service. we need to make sure that moving this log to a different log level doesn't break anything